go.mod

```diff
@@ -56,6 +57,7 @@ require (
 	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
 	github.com/modern-go/reflect2 v1.0.2 // indirect
 	github.com/munnerz/goautoneg v0.0.0-20191010083416-a7dc8b61c822 // indirect
+	github.com/opencontainers/go-digest v1.0.0 // indirect
```

Comment:
Is this used anywhere?

Reply:
This is automatically generated after `go mod tidy`. It may be a package that the `github.com/distribution/reference v0.5.0` library indirectly depends on.